PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,5 @@
 For an explanation of how to fill out the fields, please see the relevant section
-in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md)
+in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/main/PULL_REQUESTS.md)
```

Comment:
Needs to be reverted, Envoy still uses master

Reply:
Thanks, I misread that line.